data_reports/ISCN3_bibliography.bib

```diff
@@ -121,6 +132,47 @@ @techreport{Parr2006
 }
 
+@article{Ping2000,
+  title={Soil Horizon Descriptions/Classification and Lab Analysis},
+  author={Ping, L},
+  year={2000},
+  publisher={Bonanza Creek LTER}
+}
+
+@article{Ping2002,
+  title={Organic carbon stores in Alaska soils},
+  author={Ping, CL and Michaelson, GJ and Kimble, John M and Everett, LR},
+  journal={Agricultural Practices and Policies of Carbon Sequestration in Soils (eds R. Lal, JM Kimble \& R. Follet)},
+  pages={485--494},
+  year={2002}
+}
+
+@incollection{Ping2004,
+  doi = {10.1007/978-3-662-06429-0_5},
+  url = {https://doi.org/10.1007/978-3-662-06429-0_5},
+  year = {2004},
+  publisher = {Springer Berlin Heidelberg},
+  pages = {71--94},
+  author = {Chien-Lu Ping and Mark H. Clark and David K. Swanson},
+  title = {Cryosols in Alaska},
+  booktitle = {Cryosols}
+}
+
+<<<<<<< HEAD
```

Comment:
Need to remove this merge notation <<<< HEAD and ==== and >>>>. Edit the file so that this is resolved and check that the entries are not duplicated.